.prettierrc

```diff
@@ -0,0 +1,18 @@
+{
+  "printWidth": 150,
+  "tabWidth": 2,
```

Comment:
Can we call this `.prettierrc.json` to be consistent between all configuration files?

Reply:
Changed in 2fe809c